src/coreclr/crosscomponents.cmake

```
@@ -23,7 +23,7 @@ if (CLR_CMAKE_HOST_OS STREQUAL CLR_CMAKE_TARGET_OS)
     endif(CLR_CMAKE_TARGET_UNIX)
 endif()
 
-if(NOT CLR_CMAKE_HOST_LINUX AND NOT FEATURE_CROSSBITNESS)
+if(NOT CLR_CMAKE_HOST_LINUX AND NOT CLR_CMAKE_HOST_OSX AND NOT FEATURE_CROSSBITNESS)
```

Should this be something like
```suggestion
if(CLR_CMAKE_HOST_WINDOWS AND NOT FEATURE_CROSSBITNESS)
```

---

No, FreeBSD and SunOS still need to be excluded.